LICENSE

```diff
@@ -25,9 +28,3 @@ SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
 CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
 OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-
-Semver File License
```

Comment:
@bollwyvl do you know if there is a good practices to mention licenses of vendored codes? Having it mention in the README looks like reducing its visibility.

Reply:
Hi @fcollonval, I have been looking aroud and found https://github.com/orta/Heuristics-for-vendoring-MIT-code might be useful for us, probably (3) You are copying a whole file) is exactly what we want. 

Please let me know if it makes sense to you.

Reply:
Yep: the top-level `license_files` key can now be used to point to multiple files.

It should also point at the `third-party-licenses.json` file.

Reply:
Also, we should try to ensure that all of our deps _have_ licenses, and manually source them, if need be.

Reply:
What do you think about `THIRD_PARTY_LICENSES` directory?

Reply:
On the details of `pyproject.toml`, https://peps.python.org/pep-0639/#license-file-multiple-use proposes use `license-files` key but it is still draft.

Reply:
[hatch supports `license-files`](https://github.com/search?q=repo%3Apypa%2Fhatch%20license-files&type=code), and I have seen other tools like [`pyproject-fmt`](https://github.com/search?q=repo%3Atox-dev%2Fpyproject-fmt%20license-files&type=code) support it, so see no downside to it.